Tools/jit/build.py

```diff
@@ -42,12 +42,15 @@
     parser.add_argument(
         "--cflags", help="additional flags to pass to the compiler", default=""
     )
+    parser.add_argument("--llvm-version", help="LLVM version to use")
     args = parser.parse_args()
     for target in args.target:
         target.debug = args.debug
         target.force = args.force
         target.verbose = args.verbose
         target.cflags = args.cflags
+        if args.llvm_version:
```

Comment:
Can we move this to the bottom of the args list, under the pyconfig_dir line so it's easier to read?